core/src/UserSettingsEditor.html

```diff
@@ -155,6 +155,7 @@
             class="fd-input"
             type="text"
             aria-label="Image label"
+            placeholder="{schemaItem.placeholder && schemaItem.placeholder}"
```

Comment:
Why do you check it twice?

Reply:
Cause first one check is is exist and second one add value